src/quo2/components/settings/reorder_item/view.cljs

```diff
@@ -0,0 +1,43 @@
+(ns  quo2.components.settings.reorder-item.view
+  (:require [quo.react-native :as rn]
+            [react-native.platform :as platform]
+            [quo2.components.settings.reorder-item.items.item :as item]
+            [reagent.core :as reagent]
+            [quo2.components.settings.reorder-item.style :as style]
+            [quo2.components.settings.reorder-item.items.item-placeholder :as placeholder]
+            [quo2.components.settings.reorder-item.items.item-skeleton :as skeleton]
+            [quo2.components.settings.reorder-item.items.item-tabs :as tab]))
+
+
+(defn render-fn
+  [item _ _ _ _ drag]
+  (let [label-value (:label item)
+        type (:type item)
+        data (:data item)
+        default-active (:default-active item)]
+    (case type
+      "item" [item/view item drag]
```

Comment:
maybe we want to put these in a variable/const?

Reply:
Done